<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.0169492 rgba(225, 176, 50, 128), stop:1 rgba(0, 0, 0, 147));
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(133, 245, 255, 252), stop:0.35 rgba(255, 188, 188, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:0.465909 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string>FIBONACCI NUMBERS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_array">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_answer">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);
</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string>CALCULATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string>Searched Number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>441</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="widget1" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>391</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/img_185728.png</normaloff>:/icons/icons/img_185728.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
background-color: rgb(176, 255, 246);</string>
    </property>
    <property name="text">
     <string>GOLDEN RATIO</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widget1</class>
   <extends>QWidget</extends>
   <header location="global">widget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_array</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
